core/sources/domains.move

```diff
@@ -158,20 +160,30 @@ module aptos_names::domains {
     }
 
     public entry fun init_reverse_lookup_registry_v1(account: &signer) {
-        assert!(signer::address_of(account) == @aptos_names, error::permission_denied(ENOT_AUTHORIZED));
+        let addr = signer::address_of(account);
```

Comment:
Why? addr is only used once

Reply:
Fixed.  probably was from a weird iteration